data/simulations/FAH_SARS-CoV-2_Mpro_FEP.yml

```diff
@@ -0,0 +1,57 @@
+type: md
+title: Trajectories of the SARS-CoV-2 main protease protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 main protease with small molecules, simulated using Folding@Home.
+  Each small molecule was run alone in solution and docked to Mpro to compute absolute free energies of binding.
+  Simulations were run using GROMACS-5.0.4 or GROMACS-2020 and are stored as compressed binary XTC files.
+  Absolute free energies were computed using an expanded ensemble scheme, using 40 lambdas to decouple the ligand interactions.
+  Free energy data for each alchemical lambda is stored in the md.log file, and found more concisely in pre-scraped pkl dataframes.
+  The dataset comprises several projects, each having a RUN*/CLONE*/result* directory structure.
```

Comment:
You can use single backticks to protect `RUN*/CLONE*/result*` and render it as code.